flake8_django/checkers/base_model_checker.py

```diff
@@ -10,6 +10,20 @@ class BaseModelChecker(Checker):
 
     model_name_lookup = None
 
+    def is_abstract_model(self, base):
+        # look for "class Meta"
+        for elem in base.body:
+            if isinstance(elem, ast.ClassDef) and elem.name == 'Meta':
+                # look for "abstract = True"
+                for inner in elem.body:
+                    if (
+                            isinstance(inner, ast.Assign)
+                            and inner.value
+                            and any(t.id == 'abstract' for t in inner.targets)
```

Comment:
@denizdogan we have to check if `abstract` is True or False, cause if it's False, then `is_abstract_model` should return `False`.